src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -138,6 +138,14 @@
                   Visible="False" />
 
 
```

Comment:
@dsplaisted @nkolev92 Should this now be `TargetPlatformMoniker` given our discussions internally?

Reply:
Added!